api/src/main/java/org/openmrs/module/fhir2/FhirConstants.java

```diff
@@ -319,4 +319,6 @@ private FhirConstants() {
 	public static final String SNOMED_SYSTEM_URI = "http://snomed.info/sct";
 	
 	public static final String RX_NORM_SYSTEM_URI = "http://www.nlm.nih.gov/research/umls/rxnorm";
+	
+	public static final String UCUM_SYSTEM_URI = "http://unitsofmeasure.org";
```

Comment:
don't love defining these as constants in the fhir2 module. it feels like the source of truth for these system uris should be somewhere else (first thought is core openmrs?)